iati/core/tests/test_data.py

```diff
@@ -440,6 +440,7 @@ def test_detect_version_v1_simple(self, iati_tag_names, version):
         assert result == version
 
     def test_detect_version_explicit_parent_mismatch_explicit_child(self, iati_tag_names):
+        """Check that no version is detected for a v1 dataset where a version within the `iati-activities` element does not match the versions specified within all `iati-activity` child elements."""
```

Comment:
`dataset` -> `Dataset`

(here and in other docstrings)

Reply:
For an object representing a file in IATI format, I agree with `Dataset`. However should a raw file not be deemed a `dataset` though??

Reply:
Was thinking that because the test is acting on an object, it should be capitalised. Depends what the docstring is referencing, I s'pose.

Reply:
Looking at it again, I agree with `Dataset` - now fixed in 2b8303f